### PR TITLE
URW-16

### PR DESCRIPTION
**Pull request by ntindle**

### CODE REVIEW

**Comment by AustinLeath** [comment]
Is this still a relevant PR? @ntindle

**Comment by ntindle** [comment]
It can be

File: template/header.php
Comment (sebastian-king) [inline template/header.php:167]:
```suggestion
				<li class="thin auth" style="margin-left: auto;"><a href="/me/" class="stacked">My<br>Profile</a></li>

```

File: template/header.php
Comment (sebastian-king) [inline template/header.php:170]:
```suggestion
				<li class="thin" style="margin-left: auto;"><a href="/auth/login" class="stacked">Log<br>In</a></li>

```

File: template/header.php
Comment (sebastian-king) [inline template/header.php:171]:
```suggestion
	                	<li class="thin" style="margin-left: 0;"><a href="/auth/join" class="stacked">Join<br>Now</a></li>

```
We've decided to supersede the _open in new tab_ functionality so we need to remove that from the PR, otherwise looks  good

File: about.php
Comment (sebastian-king) [inline about.php:42]: the very last link seems to be empty, also we shouldn't use `./contact` -- all links should be absolute on the site so it should be `/contact`
I also can't approve the PR until you resolve the merge conflicts, because after you commit your conflict resolutions it'll take away my review :p

Comment (sebastian-king) [reply about.php:42]: bump :)

**Review by sebastian-king — CHANGES_REQUESTED** [review]
See other comments for change specifics

**Comment by ntindle** [comment]
Changes already exist in` about/index.php`.

**Pull request closed without merging**
